plugins/API/API.php

```diff
@@ -524,7 +524,7 @@ public function getBulkRequest($urls)
 
         $result = array();
         foreach ($urls as $url) {
-            $params = Request::getRequestArrayFromString($url . '&format=php&serialize=0');
+            $params = Request::getRequestArrayFromString($url . '&format=json');
```

Comment:
wouldn't we use directly the original format? Then the decode might not be needed? But I suppose this wouldn't return the same result maybe as the dataTable etc wouldn't be converted to an array 👍  Haven't checked yet but be good to test there's no change in output (doubt there is but never know)

Reply:
The JSON & XML formatter actually used the PHP Formatter to get the DataTable as an array. I have moved the method to convert the table to an array to the abstract class. So there shouldn't be any change in output when using json format an a `json_decode`